src/EEPstore.h

```diff
@@ -49,22 +49,28 @@ template <class T> class EEPstore {
   const uint16_t crc;
 
 public:
-  static void getOrDefault(T &dataRef, const uint16_t address = 0) {
+  static inline bool getIfValid(T &dataRef, const uint16_t address = 0) {
     EEPstore<T> storage(dataRef);
     EEPROM.get(address, storage);
-    if (storage.crc == storage.calcCRC()) {
+    bool valid = storage.crc == storage.calcCRC();
+    if (valid) {
       dataRef = storage.data;
-    } else {
+    }
+    return valid;
+  }
```

Comment:
This would be an alternative to `getOrDefault()` that would not implicitly write to EEPROM